src/index.d.ts

```diff
@@ -0,0 +1,37 @@
+export module "native-fetch" {
+    const _exports: {
+        default: typeof fetch;
+        Headers: {
+            new (init?: HeadersInit): Headers;
+            prototype: Headers;
+        };
+        Request: {
+            new (input: RequestInfo, init?: RequestInit): Request;
+            prototype: Request;
+        };
+        Response: {
+            new (body?: BodyInit, init?: ResponseInit): Response;
+            prototype: Response;
+            error(): Response;
+            redirect(url: string, status?: number): Response;
+        };
+    } | {
+        /**
+         * @type {typeof import("node-fetch/index").default }
+         */
+        default: typeof import("node-fetch/index").default;
+        /**
+         * @type {typeof import("node-fetch/index").Headers }
+         */
+        Headers: typeof import("node-fetch/index").Headers;
+        /**
+         * @type {typeof import("node-fetch/index").Request }
+         */
+        Request: typeof import("node-fetch/index").Request;
+        /**
+         * @type {typeof import("node-fetch/index").Response }
+         */
+        Response: typeof import("node-fetch/index").Response;
```

Comment:
Are these going to be compatible?  The `.body` property will be a node Stream which isn't compatible with the browser implementation since it's a WHATWG ReadableStream.

Reply:
Like the following comment that @type is not necessary.